### ISSUE TITLE
[Feature Request] Plugin settings popup

### ISSUE THREAD

**Issue by craciu25yt** [Vendicated/Vencord#767]
### Discord Account

craciu25_YT#3332

### What is it that you'd like to see?

It would be nice to have the plugins settings like a popup like the CSS editor. Like that if we are testing a plugin we can just click save and still testing and it will not be needed to open settings, vencord, plugins, search the plugin, click settings and edit the setting

### Request Agreement

- [X] This is not a plugin request

**Comment by Vendicated** [comment]
I don't think this is easily doable because to do this you need to start an entirely new browser session which won't have access to any of the components or state
